common/conf.py

```diff
@@ -147,7 +147,8 @@ def get_previous_version(majornumber, minornumber=0):
 oo_site_root = oo_root + '/site'
 lists_root = 'http://lists.openmicroscopy.org.uk'
 downloads_root = 'https://downloads.openmicroscopy.org'
-help_root = 'https://help.openmicroscopy.org'
+guide_root = 'https://omero-guides.readthedocs.io/en/latest/'
```

Comment:
```suggestion
guide_root = 'https://omero-guides.readthedocs.io/en/latest'
```

All URLs now include a double slash e.g. https://omero-guides.readthedocs.io/en/latest//introduction/docs/annotate.html. Not a blocker but unnecessary duplication.